omegatrade-app/frontend/src/app/components/simulation/simulation.component.html

```diff
@@ -0,0 +1,117 @@
+<app-header></app-header>
+<app-sidebar></app-sidebar>
+<div class="spinner" *ngIf="loader">
+  <mat-spinner></mat-spinner>
+</div>
+<main class="stock-dashboard">
+  <div class="container-fluid mt-20">
+    <div class="row">
+      <div class="col-6 simulate-section">
+        <span class="simulate-title">
+          Simulate Data
+        </span>
+      </div>
+      <div class="col-6 table-options">
+      </div>
+    </div>
+  </div>
+
+  <div class="container-fluid mt-20">
+    <mat-card>
+      <form class="login-form" [formGroup]="simulateForm" (submit)="simulate()">
+        <div class="row">
+          <div class="col-3 ">
+            <mat-form-field appearance="standard">
+              <mat-label>Select Company</mat-label>
+              <mat-select formControlName="companyId" id="companyId">
+                <mat-option *ngFor="let company of this.companies" [disabled]="isAlreadyStarted(company.companyId)"
+                  [value]="company.companyId">{{ company.companyName }}</mat-option>
+              </mat-select>
+            </mat-form-field>
+          </div>
+          <div class="col-3 ">
+            <mat-form-field appearance="standard">
+              <mat-label>Select Interval</mat-label>
+              <mat-select formControlName="timeInterval" id="timeInterval">
+                <mat-option *ngFor="let time of this.time" [value]="tme">{{ time }}</mat-option>
+              </mat-select>
+              <mat-hint>In seconds</mat-hint>
+            </mat-form-field>
+          </div>
+          <div class="col-3 ">
+            <mat-form-field appearance="standard">
+              <mat-label>Number of Records</mat-label>
+              <mat-select formControlName="data" id="data">
+                <mat-option *ngFor="let data of this.datas" [value]="data">{{ data }}</mat-option>
```

Comment:
`*ngFor="let data of this.datas"`
Instead of **datas** can we provide some meaningful name? datas would not be understandable, for ex. noOfRecordsList, but you also can think for other names but it should be meaningful

Reply:
the above suggestion has been added